docs/platform/concepts/byoc.rst

```diff
@@ -1,18 +1,19 @@
-Bring your own cloud (BYOC)
-===========================
+Bring your own cloud (BYOC) |beta|
+==================================
 
-Aiven services are usually deployed on Aiven managed infrastructure, using
-Aiven managed security protocols, and backed by Aiven managed storage and backups.
-This provides the most seamless, straight forward, and de-risked approach to deploying
-Aiven services. However, there are cases where this approach is not appropriate, such
-as the need to achieve strict regulatory compliance.
+.. topic:: Availability of BYOC
 
-In cases like these, Aiven offers customers the ability to instead BYOC (Bring
-Your Own Cloud).  BYOC allows customers to manage their own infrastructure,
-their own security posture and keep their data in their own cloud.
+   This is a beta feature, and its availability is limited. To enable BYOC for your Aiven organization, contact `sales@Aiven.io <mailto:sales@Aiven.io>`_.
```

Comment:
From my point of view, BYOC is not beta (we have several customers in production that way).
The beta part is the self service

Reply:
that's correct, @Matthieu68857 ! Thanks for spotting that.